getting-started/edit-escrow-hints.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Hints
+order: 9
```

Comment:
I originally tried to hide this from the sidebar, so it only appears when you link to it from the work page. But I guess no real need to do so.